Geometry/CaloTopology/test/HcalTopologyTester.cc

```diff
@@ -2,18 +2,61 @@
 #include <string>
 #include <vector>
 
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESTransientHandle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+#include "Geometry/Records/interface/HcalRecNumberingRecord.h"
+#include "Geometry/HcalCommonData/interface/HcalDDDRecConstants.h"
 #include "Geometry/CaloTopology/interface/HcalTopology.h"
 #include "DataFormats/HcalDetId/interface/HcalDetId.h"
 #include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+#include "CoralBase/Exception.h"
```

Comment:
Removed dependency
________________________________
From: David Lange [notifications@github.com]
Sent: 26 September 2015 13:54
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx28 Make the topology tester up-to-date with the geometry changes for HCAL (#11423)


In Geometry/CaloTopology/test/HcalTopologyTester.cc<https://github.com/cms-sw/cmssw/pull/11423#discussion_r40492941>:

>  #include "Geometry/CaloTopology/interface/HcalTopology.h"
>  #include "DataFormats/HcalDetId/interface/HcalDetId.h"
>  #include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
> +#include "CoralBase/Exception.h"


why is this dependency here? (and in the buildfile?)

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11423/files#r40492941>.